pyseir/load_data.py

```diff
@@ -301,6 +303,21 @@ def load_county_metadata_by_fips(fips):
     return county_metadata_merged
 
 
+@lru_cache(maxsize=32)
+def get_all_fips_codes_for_a_state(state: str):
+    """Returns a list of fips codes for a state
+
+    Arguments:
+        state {str} -- the full state name
+
+    Returns:
+        fips [list] -- a list of fips codes for a state
+    """
+    df = load_county_metadata()
```

Comment:
Can we use `libs.datasets.combined_datasets.build_us_latest_with_all_fields()` for this so that the fips codes match the rest of the input data we have?

Reply:
could we do this in a separate pr?

Reply:
sounds good!